File: scryfall/card_symbol.go
package scryfall

type CardSymbol struct {
	Symbol             string   `json:"symbol"`
	LooseVariant       string   `json:"loose_variant,omitempty"`
	English            string   `json:"english"`
	Transposable       bool     `json:"transposable"`
	RepresentsMana     bool     `json:"represents_mana"`
	CMC                float64  `json:"cmc,omitempty"`
	AppearsInManaCosts bool     `json:"appears_in_mana_costs"`
	Funny              bool     `json:"funny"`
	Colors             Colors   `json:"colors"`
	GathererAlternates []string `json:"gatherer_alternates,omitempty"`
}

File: packages/scryfall/src/types/card_symbol.ts
import {Colors} from './color';
import {URI} from './scryfall';

export type CardSymbol = {
    object: 'card_symbol';
    symbol: string;
    loose_variant?: string;
    english: string;
    transposable: boolean;
    represents_mana: boolean;
    cmc?: number;
    appears_in_mana_costs: boolean;
    funny: boolean;
    colors: Colors;
    gatherer_alternates?: string[];
    svg_uri?: URI;
};

export function isCardSymbol(o: unknown): o is CardSymbol {
    return Boolean(o && typeof o === 'object' && (o as any).object === 'card_symbol');
}
